storage/cloudspanner/log_storage.go

```diff
@@ -681,18 +696,53 @@ func (tx *logTX) GetLeavesByIndex(ctx context.Context, indices []int64) ([]*tril
 		return nil, err
 	}
 
-	leaves := make(leafmap)
 	stmt := spanner.NewStatement(
-		`SELECT sd.MerkleLeafHash, ld.LeafValue, ld.ExtraData, sd.SequenceNumber, ld.LeafIdentityHash, ld.QueueTimestampNanos, sd.IntegrateTimestampNanos
-FROM SequencedLeafData as sd
-INNER JOIN LeafData as ld
-ON sd.TreeID = ld.TreeID AND sd.LeafIdentityHash = ld.LeafIdentityHash
-WHERE sd.TreeID = @tree_id and sd.SequenceNumber IN UNNEST(@seq_nums)`)
+		`SELECT 
```

Comment:
Could you add a comment somewhere elaborating why you don't use JOIN? Seems like the reason is that the cloudspanner testing tool does not support JOIN, while the production cloudspanner does. Would we want to switch back to JOIN when the testing tool supports it? In this case I would also add a TODO.

Reply:
In a TODO please add the link where the testing tool status lists the unsupported features.

Reply:
done

Reply:
Not done?

Reply:
Missed the TODO, added it to the next PR